### PR TITLE
F1 - Landing page added

### PR DESCRIPTION
**Pull request by sushmita12345**
Landing page is added to E-commerce website
Screenshot 2022-03-16 at 5 21 08 PM
.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,41 +1,12 @@
 import "./App.css";
-import logo from "./logo.png";
+import {Home} from './pages/HomePage/Home';
+// import {Navbar} from './components/Navbar/navbar';
+// import {Header} from './components/Header/Header';
```
Comment (PuneetP16) [inline src/App.js:4]: remove unused comments

File: src/components/Filter/Sidebar.jsx [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+      <div className="sidebar-content-2">
+        <h3 className="content-1a">Sort by</h3>
+        <div className="sidebar-content-4a">
+          <div className="radio-content">
+            <input
+              type="radio"
+              value="low"
+              name="group"
+              onChange={() =>
+                filterDispatch({ type: "SORT_BY_PRICE", payload: "lowToHigh" })
```
Comment (PuneetP16) [inline src/components/Filter/Sidebar.jsx:165]: You could add type: "SORT_BY_PRICE" in a constant to avoid typo error and for ease of reuse.

**Review by PuneetP16 — APPROVED** [review]
Code looks good to merge.

**Review by devsurajrai — APPROVED** [review]
Everything looks good. Good to go.

**Pull request merged**